core/src/main/java/io/parsingdata/metal/data/ParseGraph.java

```diff
@@ -119,11 +119,11 @@ ParseGraph closeBranch() {
 
     @Override
     public String toString() {
-        if (this == EMPTY) { return "graph(EMPTY)"; }
+        if (this == EMPTY) { return "pg(EMPTY)"; }
```

Comment:
Is the `p` really needed? Could be shorter, so instead `pg`, `pref`, `pval`, maybe,  `gr`, `ref`, `val`?